os/drivers/testcase/testcase_drv.c

```diff
@@ -22,8 +22,14 @@
 #include <tinyara/config.h>
 #include <errno.h>
 #include <debug.h>
+#include <unistd.h>
+#include <time.h>
 #include <tinyara/fs/fs.h>
 #include <tinyara/testcase_drv.h>
+#include <tinyara/sched.h>
+#include "../kernel/clock/clock.h"
```

Comment:
Don't use relative path to include headers.
Please use CFLAGS.
```
  CFLAGS += -I$(TOPDIR)/kernel
```

Reply:
I added it and modified.